db/migrate/20170605090919_add_expression_type_to_miq_alert.rb

```diff
@@ -0,0 +1,23 @@
+class ::MiqExpression; end
```

Comment:
Thanks!  We suspected so but weren't sure what's a right way.
It's a serialized column, without a stub accessing it will instantiate lib/miq_expression.rb
(uncaught by good_migrations BTW, though it does fail when that (wrongly?) leads to requiring app/models/tag.rb.)
And namespacing the stub inside the migration `serialize` doesn't pick it.  Is there a way to deserialize using custom class namespace?
Or manipulate the YAML with regexps like https://github.com/ManageIQ/manageiq/blob/master/db/migrate/20150625220141_fix_serialized_reports_for_rails_four.rb ?